lib/KmsClient.js

```diff
@@ -700,19 +701,19 @@ export class KmsClient {
 
       const agent = httpsAgent || this.agent;
       // send request
-      const result = await httpClient.post(url, {
+      result = await httpClient.post(url, {
         agent, headers, json: config
       });
-
-      _assert(result.data, 'result.data', 'object');
-      _assert(result.data.id, 'result.data.id', 'string');
-      return result.data;
     } catch(e) {
       _handleClientError({
```

Comment:
I presume this throws... maybe a comment.  The helper would be better named `_throwClientError`, the lack of a clear understanding makes the lack of a `return` concerning.

Reply:
Created an issue.